recipes/flit/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Build might be run as root, but it doesn't matter
+set -e
+
+export FLIT_ROOT_INSTALL=1
+python -m flit install --env --deps none
+
+which python
+cd && python -c "import flit; print(flit)"
```

Comment:
You don't need the `set -e` and `which python` in the recipe. Specially if you use `$PYTHON` here :wink: